src/mmrelay/db_runtime.py

```
@@ -526,12 +538,24 @@ def close(self) -> None:
                     )
 
-        if hasattr(self._thread_local, "connection"):
+        old_thread_local = self._thread_local
+        self._thread_local = threading.local()
+
+        if hasattr(old_thread_local, "connection"):
             try:
-                del self._thread_local.connection
+                del old_thread_local.connection
             except AttributeError:
                 pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(AttributeError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(AttributeError): ...`

(SIM105)